README.md

```diff
@@ -5,68 +5,34 @@
-
 ## Overview
 
-[Introduction video](https://www.youtube.com/watch?v=AmdKF2UhFzw)
-
-SystemJS provides two hookable base builds:
-
-#### 1. s.js minimal loader
+SystemJS is a hookable standards-based module loader. It provides a workflow where code written for production workflows of native ES modules in browsers ([like Rollup code-splitting builds](https://rollupjs.org/guide/en#code-splitting)), can be transpiled to the [System.register module format](docs/system-register.md) to work in older browsers that don't support native modules, running [almost-native module speeds](#performance) while supporting top-level await, dynamic import, circular references and live bindings, import.meta.url, module types, import maps, integrity and Content Security Policy with compatibility in older browsers back to IE11.
 
-The minimal [2.3KB s.js loader](dist/s.min.js) provides a workflow where code written for production workflows of native ES modules in browsers ([like Rollup code-splitting builds](https://rollupjs.org/guide/en#code-splitting)), can be transpiled to the [System.register module format](docs/system-register.md) to work in older browsers that don't support native modules, including [IE11](#ie11-support).
+#### 1. s.js minimal production loader
 
-Since the ES module semantics such as live bindings, circular references, contextual metadata, dynamic import and top-level await [can all be fully supported this way](docs/system-register.md#semantics), while supporting CSP and cross-origin support, this workflow can be relied upon as a polyfill-like path.
+The minimal [2.8KB s.js production loader](dist/s.min.js) includes the following features:
 
-* Support for loading [bare specifier names](docs/import-maps.md) through import maps (formerly package maps, formerly map configuration), loaded via `<script type="systemjs-importmap">`.
-* Loads and resolves modules as URLs, throwing for bare specifier names (eg `import 'lodash'`) like the native module loader.
-* Loads System.register modules.
-* Core hookable extensible loader supporting [custom extensions](docs/hooks.md).
+* Loads `System.register` modules, the CSP-compatible [SystemJS module format](docs/system-register.md).
+* Support for loading bare specifier names with [import maps](docs/import-maps.md) via `<script type="systemjs-importmap">`.
+* Supports [hooks](docs/hooks.md) for loader customization.
 
 #### 2. system.js loader
 
-The [3.7KB system.js loader](dist/system.min.js) loader builds on the s.js core and adds support for [global loading](#extras), [non-javascript module](/docs/module-types.md), and the [SystemJS registry API](/docs/api.md#registry).
+The [4.2KB system.js loader](dist/system.min.js) adds the following features in addition to the `s.js` features above:
```

Comment:
Are we planning on removing the `dist` directory from source control in the 7.0 release?

Reply:
Yeah that would be nice to do, when we get to that release.